Code/Framework/AzToolsFramework/AzToolsFramework/ToolsComponents/ScriptEditorComponent.cpp

```diff
@@ -1058,6 +1058,7 @@ namespace AzToolsFramework
                             Attribute(AZ::Edit::Attributes::NameLabelOverride, &AzFramework::ScriptPropertyGroup::m_name)->
                             Attribute(AZ::Edit::Attributes::AutoExpand, true)->
                         DataElement(nullptr, &AzFramework::ScriptPropertyGroup::m_properties, "m_properties", "Properties in this property group")->
+                            Attribute(AZ::Edit::Attributes::ContainerCanBeModified, false)->
```

Comment:
Is this DPE related or was this container mutable in an undesirable way in the RPE too?

Reply:
the RPE had a special case to keep it from being mutable, but this is more generic.